docs/pages/x/api/charts/charts-axis.json

```diff
@@ -37,7 +37,24 @@
     "import { ChartsAxis } from '@mui/x-charts/ChartsAxis';",
     "import { ChartsAxis } from '@mui/x-charts';"
   ],
-  "styles": { "classes": [], "globalClasses": {}, "name": "MuiChartsAxis" },
+  "styles": {
+    "classes": [
```

Comment:
This change looks strange to me. Doesn't looks like this component actually has these slots. 🙈 
@alexfauquette could you take a look?

Reply:
Clarification, the generated change seems correct given the existing themeAugmentation declaration, but the registered slots do not seem to be consumed and/or applied anywhere... 🤷

Reply:
The classes exist, but the `styleOverride` don't if that's what you mean. But it's a problem on `x-charts` side

Reply:
I've looked more deeply into this matter and my initial assumption was incorrect.
This is actually a needed fix.
But I believe I see a different problem regarding `MuiChartsXAxis` and `MuiChartsYAxis`.

Reply:
Fine from my PoV. I've prepared a PR removing the not working slots: https://github.com/mui/mui-x/pull/9996